app/javascript/components/lists/ListButtons.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState} from "react"
+
+const ListButtons = (props) => {
+  const [showEditButton, setShowEditButton] = React.useState(0)
+
+  const csrf = props.csrf;
+  const list = props.list;
+
+  let buttonGroup;
+  if( showEditButton == 0 ){
+    buttonGroup = 
+    <React.Fragment>
+      <div className="btn-group" role="group">
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}></button>
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}></button>
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}></button>
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}>Edit</button>
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}></button>
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}></button>
+        <button className="btn btn-warning" onClick={() => setShowEditButton(1)}></button>
```

Comment:
I'm confused about why there are 7 buttons.  The PR body says it's to do with consistent sizing, but could we instead use a CSS class that sets the appropriate width, and only have one button?

Reply:
Indeed, I will look into making that change on this PR today.

Reply:
Fixed